### PR TITLE
Add card story plus styling

### PR DESCRIPTION
**Pull request by mwarnck**

### CODE REVIEW

File: styles/components/card.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
+.card {
+  border: 1px solid black;
+  border-radius: 10px;
+  position: relative;
+  background-color: lightgoldenrodyellow;
+  height: max-content;
+  max-width: 400px;
+  margin: 30px auto;
+  xdisplay: flex;
```
Comment (Quaquaro) [inline styles/components/card.css]: Maybe you forget to remove the x or is this for later?

File: styles/components/card.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  background-color: lightgrey;
+  border-radius: 10px;
+}
+
+.card__summary:hover {
+  background-color: lightsalmon;
+  color: white;
+}
+
+.card__ul {
```
Comment (Quaquaro) [inline styles/components/card.css:43]: I believe you can put the list-style-none attribute from below here

Comment (Quaquaro) [inline styles/components/card.css:43]: For the accessibility you can you can use role="list" in your HTML-file and .card__ul[role="list"] in your CSS-file

File: index.html [hunk 3]
```diff
@@ -7,39 +7,74 @@
...
+          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Modi earum
+          voluptas quisquam numquam aut nostrum, voluptatum voluptatibus nobis.
+        </p>
+        <details>
+          <summary class="card__summary">Show answer</summary>
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat,
+          sed diam voluptua.
+        </details>
+        <ul class="card__ul">
```
Comment (Quaquaro) [inline index.html]: Why you wanna hide the answers?
give the ul a role="list" for accessibility.

File: styles/components/card.stories.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+export default {
+  title: "Components/card",
```
Comment (Quaquaro) [inline styles/components/card.stories.js]: May you write card with a Capital C to follow the Convention

Comment (mwarnck) [reply styles/components/card.stories.js]: yes thanks you are right

File: index.html (hunk 3 above)
Comment (mwarnck) [reply index.html]: isn´t it part of the app that the answer appear and disappear by clicking the button?

**Pull request merged**